Spigot-Server-Patches/0386-Implement-getters-and-setters-for-EntityItem-owner-a.patch

```diff
@@ -0,0 +1,89 @@
+From 913a7fcf279714090ea071933c2c0307ee426527 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Sat, 6 Oct 2018 20:54:23 -0500
+Subject: [PATCH] Implement getters and setters for EntityItem owner and
+ thrower
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityItem.java b/src/main/java/net/minecraft/server/EntityItem.java
+index f596b4351..f21a5915a 100644
+--- a/src/main/java/net/minecraft/server/EntityItem.java
++++ b/src/main/java/net/minecraft/server/EntityItem.java
+@@ -410,20 +410,24 @@ public class EntityItem extends Entity {
+         this.getDataWatcher().set(EntityItem.b, itemstack);
+     }
+ 
++    public UUID getOwner() { return k(); } // Paper - OBFHELPER
+     @Nullable
+     public UUID k() {
```

Comment:
Maybe a Paper comment on this line to force a conflict if the method name is changed?

Reply:
This should conflict as-is, so non-issue here :)